prereise/gather/hiflddata/data_trans.py

```diff
@@ -0,0 +1,569 @@
+    print(missingSub[:20])
+
+
+def Write_bus2sub(clean_data):
+    """Write the data to bus2sub.csv as output
+
+    :param pandas.DataFrame clean_data: substation dataframe as returned by :func:`Clean`
+    """
+
+    with open('output/bus2sub.csv','w',newline='') as bus2sub:
+        csv_writer = csv.writer(bus2sub)
+        csv_writer.writerow(["Bus_id","sub_id"])
+        for index, row in clean_data.iterrows():
+            csv_writer.writerow([row['ID'],row['ID']])
+
+
+def Write_branch(lines):
+    """Write the data to branch.csv as output
+
+    :param list lines:  a list of lines as returned by :func:`Neighbors`
+    """
+
+    with open('output/branch.csv', 'w', newline='') as branch:
+        csv_writer = csv.writer(branch)
+        csv_writer.writerow(['branch_id', 'line_type', 'from_bus_id', 'from_bus_name', 'to_bus_id', 'to_bus_name', 'length_in_mile', 'reactance', 'rateA'])
+        csv_writer.writerows(lines)
+
+
+# Reactance Calculate 1
+kv_xperunit_calculate_1 = (
+    (69, 0.2293),
+    (115, 0.1106),
+    (230, 0.0567),
+    (345, 0.0074),
+    (500, 0.0001),
+    (1100, 0.0001)
+)
+
+# Reactance Calculate 2
+kv_xperunit_calculate_2 = (
+    (69, 0.0096),
+    (100, 0.0063),
+    (115, 0.0039),
+    (138, 0.0026),
+    (161, 0.0021),
+    (230, 0.0011),
+    (345, 0.0005),
+    (500, 0.0002),
+    (765, 0.0001),
+    (1100, 0.0001)
+)
+
+# RateA Calculate 3
+kv_rate_A_calucate_3 = (
+    (69, 86),
+    (100, 181),
+    (115, 239),
+    (138, 382),
+    (161, 446),
+    (230, 797),
+    (345, 1793),
+    (500, 2598),
+    (765, 5300),
+    (1100, 5300)
+)
+
+# RateA Calculate 3
+kv_sil_calucate_3 = (
+    (69, 13),
+    (100, 30),
+    (115, 35),
+    (138, 50),
+    (161, 69),
+    (230, 140),
+    (345, 375),
+    (500, 1000),
+    (765, 2250),
+    (1100, 2250)
+)
+
+# Reactance Calculate 4
+kv_from_to_xperunit_calculate_4 = {
+    (69, 165): 0.14242,
+    (69, 138): 0.10895,
+    (69, 161): 0.14943,
+    (69, 230): 0.09538,
+    (69, 345): 0.08896,
+    (115, 161): 0.04516,
+    (115, 230): 0.04299,
+    (115, 345): 0.04020,
+    (115, 500): 0.06182,
+    (138, 161): 0.02818,
+    (138, 230): 0.03679,
+    (138, 345): 0.03889,
+    (138, 500): 0.03279,
+    (138, 765): 0.02284,
+    (161, 230): 0.06539,
+    (161, 345): 0.03293,
+    (161, 500): 0.06978,
+    (230, 345): 0.02085,
+    (230, 500): 0.01846,
+    (230, 765): 0.01616,
+    (345, 500): 0.01974,
+    (345, 765): 0.01625,
+    (500, 765): 0.00436,
+}
+
+def computeReactanceAndType(line_type, KV_from, KV_to, dist, vol):
+    """Compute the reactance based on the type of each transmission line
+
+    :param str line_type: type of the transmission line between two substations
+    :param int KV_from: KV of the substation1 at one end
+    :param int KV_to: KV of the substation2 at the other end
+    :param float dist: distance between the two substations
+    :param int vol: transmission line voltage
+    :return: (*float*) --  reactance value.
```

Comment:
It seem that the function actually has two return values?

Reply:
will add doc string for another return value in next revision.